src/dag-node/toDagLink.js

```diff
@@ -1,16 +1,14 @@
 'use strict'
 
 const DAGLink = require('../dag-link/dagLink')
-const { serializeDAGNode } = require('../serialize')
 const genCid = require('../genCid')
 
 /*
  * toDAGLink converts a DAGNode to a DAGLink
  */
 const toDAGLink = async (node, options = {}) => {
-  const serialized = serializeDAGNode(node)
-  const nodeCid = await genCid.cid(serialized)
-  return new DAGLink(options.name || '', serialized.length, nodeCid)
+  const nodeCid = await genCid.cid(node.serialize(), options)
```

Comment:
oh yeah, passing `options` here is a new feature, is it worth mentioning? basically I wanted to be able to get v0 CIDs out of a `addLink(node)` instead of going indirectly through `util.cid()`.